fix(bootstrap4-theme): fixed controls not showing on certain components

davidornelas11 opened the pull request.

davidornelas11 committed:
fix(bootstrap4-theme): fixed controls not showing on certain components

mlsamuelson reviewed (changes requested):
@davidornelas11 I noticed you rolled this out to some but not all `examples` and `stories`. I'm presuming those that had controls and were potentially affected.  Can we make the file naming consistent across all stories in the `bootstrap4-theme` so we avoid triggering this issue in the future if we add controls to a story or if someone is making a copy of a story but doesn't understand the implications of the naming?

davidornelas11 committed:
fix(bootstrap4-theme): added 'stories' to file names to fix some controls not showing

davidornelas11 committed:
fix(component-carousel): modified imports on stories

modified imports on stories in image and card carousel stories to accout for new file renames

davidornelas11 committed:
Merge branch 'dev' into UDS-1123-stories-not-showing-up

davidornelas11 committed:
Merge remote-tracking branch 'origin' into UDS-1123-stories-not-showing-up

mlsamuelson reviewed (changes requested):
I'm seeing path resolution errors when attempting to run `yarn storybook` in the `bootstrap4-theme` package. Can you resolve those?

davidornelas11 committed:
fix(bootstrap4-theme): fixed path errors when starting storybook

davidornelas11 committed:
Merge remote-tracking branch 'origin' into UDS-1123-stories-not-showing-up

davidornelas11 committed:
fix(component-carousel): fixed imports for stories in carousel

mlsamuelson reviewed (changes requested):
@davidornelas11 This one is going to require some conflict resolutions after the build updates.

davidornelas11 committed:
Merge branch 'dev' into UDS-1123-stories-not-showing-up

davidornelas11 committed:
Merge branch 'dev' into UDS-1123-stories-not-showing-up

mlsamuelson commented:
@davidornelas11 New conflict to resolve on this one. 😭  Then let's finally get this one in! 🥲

mlsamuelson reviewed (changes requested):
One (hopefully) final conflict resolution.

davidornelas11 committed:
Merge branch 'dev' into UDS-1123-stories-not-showing-up
